app/src/main/java/com/github/sdp/ratemyepfl/database/reviewable/ClassroomRepositoryImpl.kt

```diff
@@ -0,0 +1,64 @@
+package com.github.sdp.ratemyepfl.database.reviewable
+
+import com.github.sdp.ratemyepfl.database.Repository
+import com.github.sdp.ratemyepfl.database.query.Query
+import com.github.sdp.ratemyepfl.database.query.QueryResult
+import com.github.sdp.ratemyepfl.database.reviewable.ReviewableRepositoryImpl.Companion.AVERAGE_GRADE_FIELD_NAME
+import com.github.sdp.ratemyepfl.database.reviewable.ReviewableRepositoryImpl.Companion.NUM_REVIEWS_FIELD_NAME
+import com.github.sdp.ratemyepfl.model.items.Classroom
+import com.github.sdp.ratemyepfl.model.review.ReviewRating
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.ktx.getField
+import javax.inject.Inject
+
+class ClassroomRepositoryImpl(val repository: ReviewableRepositoryImpl<Classroom>) :
```

Comment:
Can you come back to the old names, our convention was : Interface in the name for interfaces. And not Impl in the name for implementations.

Reply:
I changed the names to follow the Kotlin naming convention for delegation. I think it is easier for an outsider to understand what's going on.